velox/type/Variant.h

```diff
@@ -301,6 +347,40 @@ class variant {
     return {TypeKind::OPAQUE, new detail::OpaqueCapsule{type, input}};
   }
 
+  static variant shortDecimal(
+      const std::optional<int64_t> input,
+      const TypePtr& type) {
+    VELOX_CHECK(type->isShortDecimal(), "Not a ShortDecimal type");
+    auto decimalType = type->asShortDecimal();
+    if (input.has_value()) {
+      return {
+          TypeKind::SHORT_DECIMAL,
+          new detail::ShortDecimalCapsule{
+              ShortDecimal(input.value()),
+              decimalType.precision(),
+              decimalType.scale()}};
+    } else {
+      return {TypeKind::SHORT_DECIMAL, nullptr};
```

Comment:
Do we need to make some changes to block usage of `variant::null(TypeKind::SHORT_DECIMAL)`?

```
  static variant null(TypeKind kind) {
    return variant{kind};
  }
```

Reply:
Good catch! I missed this.